xbmc/cores/VideoPlayer/VideoPlayerVideo.cpp

```diff
@@ -900,6 +902,9 @@ int CVideoPlayerVideo::OutputPicture(const DVDVideoPicture* src, double pts)
     return EOS_DROPPED;
   }
 
+  m_processInfo.SetVideoDimensions(pPicture->iWidth, pPicture->iHeight);
```

Comment:
Those methods are not supposed to be called on every cycle. Dimensions certainly won't change in the output stage